man/passwd.1.xml

```diff
@@ -229,7 +230,7 @@
 	    still be able to login using another authentication token
 	    (e.g. an SSH key). To disable the account, administrators
 	    should use <command>usermod --expiredate 1</command> (this sets
-	    the account's expire date to 1970-01-02).
+	    the account's expire date to 1970-01-02 00:00:00 UTC).
```

Comment:
This is a date and time, not a date.  Since --expiredate uses dates, I'd continue talking about dates, and omit the time.  However, I'd mention the timezone, so `1970-01-02 UTC`.